test/tests-psc/app.toml

```diff
@@ -61,16 +66,16 @@ start = true
 task-slots = ["sys"]
 
 [tasks.i2c_driver.interrupts]
-"i2c2.event" = 0b0000_0010
-"i2c2.error" = 0b0000_0010
-"i2c3.event" = 0b0000_0100
-"i2c3.error" = 0b0000_0100
-"i2c4.event" = 0b0000_1000
-"i2c4.error" = 0b0000_1000
+"i2c2.event" = "0b0000_0010"
```

Comment:
?

Reply:
this whole test app has bitrotted due to IRQs having names now, I changed these just to get it to compile so I could see if the kernel size was good.